controls/def.cf

```diff
@@ -126,6 +126,14 @@ bundle common def
                           not(isvariable("trustkeysfrom"))),
         meta => { "defvar" };
 
+      "cf_serverd_maxconnections"
```

Comment:
I've been thinking about this as well.

I think there is value in being able to have policy that determines a setting if no user supplied value is given.

I wonder if it might make sense to have a cfengine:controls bundle get populated with special control settings, and that could be handled like bundle common def is today. If no bundle is present the settings from the augments file are used. If there is a bundle, then the policy can decide what is best.